_data/shared_hosts.yml

```diff
@@ -236,6 +236,22 @@
   manual_update: "Yes"
   auto_update: "No"
 
+- name: "Seeweb (Cloud Server)"
+  url: "http://www.seeweb.it/"
+  php53: 29
+  default: 5.3.29
+  manual_update: "No"
+  auto_update: "Yes (Patch)"
+
+- name: "Servage"
```

Comment:
I'm trying to avoid adding any more totally unknown ones. Could you find this out, or remove it please?